File: app/src/main/java/com/maodq/ffmpegdemo/java/Constant.java
package com.maodq.ffmpegdemo.java;

import android.os.Environment;

public class Constant {
    public static final String SRC_0 = "test.wav";
    public static final String SRC_1 = "test.mp4";

    public static final String fileParent = Environment.getExternalStorageDirectory().getAbsolutePath() + "/" + "ffmpeg";

    public static final String src_0 = fileParent + "/" + SRC_0;
    public static final String src_1 = fileParent + "/" + SRC_1;
    public static final String output = fileParent + "/" + "output.aac";
}

File: app/src/main/java/com/maodq/ffmpegdemo/Constant.kt
package com.maodq.ffmpegdemo

import android.os.Environment

class Constant {
    companion object {

        const val SRC_0 = "test.wav"
        const val SRC_1 = "test.mp4"

        var fileParent: String = Environment.getExternalStorageDirectory().absolutePath + "/" + "ffmpeg"

        var src_0: String = "$fileParent/$SRC_0"
        var src_1: String = "$fileParent/$SRC_1"
        var output: String = "$fileParent/output.aac"
    }
}